pkg/agent/openflow/pipeline.go

```diff
@@ -678,17 +678,29 @@ func (c *client) ctRewriteDstMACFlows(gatewayMAC net.HardwareAddr, category cook
 	return flows
 }
 
-// serviceLBBypassFlow makes packets that belong to a tracked connection bypass
+// serviceLBBypassFlows makes packets that belong to a tracked connection bypass
 // service LB tables and enter egressRuleTable directly.
-func (c *client) serviceLBBypassFlow(ipProtocol binding.Protocol) binding.Flow {
+func (c *client) serviceLBBypassFlows(ipProtocol binding.Protocol) []binding.Flow {
 	connectionTrackStateTable := c.pipeline[conntrackStateTable]
-	return connectionTrackStateTable.BuildFlow(priorityNormal).MatchProtocol(ipProtocol).
+	var flows []binding.Flow
```

Comment:
nitpicker: looks like we can directly construct the return value rather than append? like:
```go
return []binding.Flow{
}
```

Reply:
done